server/app/assets/javascripts/toast.ts

```diff
@@ -35,6 +35,7 @@ class ToastController {
       'top-0',
       'transform',
       '-translate-x-1/2',
+      'z-20',
```

Comment:
Why add z-20 in the screen height PR?

Reply:
I had to add z indices to make things work. When I made body scrollable - text of some elements in body started to be on top of the header during scrolling. To fix it I made z-index of the header to be 10. But then I need to make toast messages to be on top of the header so adding z-20 here.